detectable/src/main/java/com/synopsys/integration/detectable/detectables/bitbake/collect/BuildFileFinder.java

```diff
@@ -50,9 +50,22 @@ public File findTaskDependsFile(File sourceDir, File buildDir) throws Integratio
 
     public Optional<File> findLicenseManifestFile(File buildDir, String targetImageName, BitbakeEnvironment bitbakeEnvironment) {
         try {
+            String machine = bitbakeEnvironment.getMachine().orElse(null);
+
             String machineArch = bitbakeEnvironment.getMachineArch().orElse(null);
             File licensesDir = findLicensesDir(buildDir, bitbakeEnvironment.getLicensesDirPath().orElse(null));
-            logger.debug("Checking licenses dir {} for license.manifest for {}", licensesDir.getAbsolutePath(), targetImageName);
+
+            logger.debug(
+                "Checking licenses dir {} for license.manifest for image {}, architecture {}, and machine {}",
+                licensesDir.getAbsolutePath(), targetImageName, machineArch, machine
+            );
+
+            Optional<File> licenseFile = findMostRecentLicenseManifestFileForMachine(licensesDir, machineArch, targetImageName, machine);
```

Comment:
First attempt to find the manifest according the "newest" format.

Reply:
Understood, thanks for the clarification.